docs/advtool-installation.html

```diff
@@ -64,50 +65,86 @@ <h4 class="ibm-h4">Taking advantage of leading edge tools and libraries</h4>
     <tr>
       <td>Debian 8</td>
       <td></td>
-      <td><p class="ibm-ind-link"><a class="ibm-anchor-down-link" href="#installUbuntu">Install Advance Toolchain</a></p><br>
-      <p class="ibm-ind-link"><a class="ibm-forward-link" href="https://developer.ibm.com/linuxonpower/advance-toolchain/advtool-compiler">Install cross compiler</a></p><br>
+      <td></td>
+      <td><p class="ibm-ind-link"><a class="ibm-anchor-down-link" href="#installUbuntu">Install Advance Toolchain</a></p><br />
+      <p class="ibm-ind-link"><a class="ibm-forward-link" href="https://developer.ibm.com/linuxonpower/advance-toolchain/advtool-compiler">Install cross compiler</a></p><br />
       <p class="ibm-ind-link"><a class="ibm-btn-sec ibm-btn-gray-50 ibm-forward-link" href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/debian/dists/jessie/at10.0">Repository and release notes</a></p></td>
     </tr>
     <tr>
       <td>Debian 9</td>
-      <td><p class="ibm-ind-link"><a class="ibm-anchor-down-link" href="#installUbuntu">Install Advance Toolchain</a></p><br>
-      <p class="ibm-ind-link"><a class="ibm-forward-link" href="https://developer.ibm.com/linuxonpower/advance-toolchain/advtool-compiler">Install cross compiler</a></p><br>
-      <p class="ibm-ind-link"><a class="ibm-btn-sec ibm-btn-gray-50 ibm-forward-link" href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/debian/dists/stretch/at11.0">Repository and release notes</a></p></td>
+      <td>
+        <p class="ibm-ind-link"><a class="ibm-anchor-down-link" href="#installUbuntu">Install Advance Toolchain</a></p><br />
+        <p class="ibm-ind-link"><a class="ibm-forward-link" href="https://developer.ibm.com/linuxonpower/advance-toolchain/advtool-compiler">Install cross compiler</a></p><br />
+        <p class="ibm-ind-link"><a class="ibm-btn-sec ibm-btn-gray-50 ibm-forward-link" href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/debian/dists/stretch/at12.0">Repository and release notes</a></p>
+      </td>
+      <td>
+        <p class="ibm-ind-link"><a class="ibm-anchor-down-link" href="#installUbuntu">Install Advance Toolchain</a></p><br />
+        <p class="ibm-ind-link"><a class="ibm-forward-link" href="https://developer.ibm.com/linuxonpower/advance-toolchain/advtool-compiler">Install cross compiler</a></p><br />
+        <p class="ibm-ind-link"><a class="ibm-btn-sec ibm-btn-gray-50 ibm-forward-link" href="ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/debian/dists/stretch/at11.0">Repository and release notes</a></p>
+      </td>
       <td></td>
```

Comment:
another extra empty cell.Notice that we don't have those on the other sections.

Reply:
AT 10.0 does not support Debian 9.